docs/.vuepress/public/ol-logo-color.svg

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="50px" height="50px" viewBox="0 0 50 50" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 51.2 (57519) - http://www.bohemiancoding.com/sketch -->
```

Comment:
Running this file through svgo (or the web version at svgomg) is likely to make some minimal size improvements, removing extraneous comments like this and some other no-op markup. [FWIW generally I’d prefer to keep UI changes like this in a different PR so they can be discussed separately, if they don’t have bearing on the content changes.]

Reply:
Thanks for the tip. Replaced this file after running through svgomg.